assembl/static2/js/app/actions/authenticationActions.js

```diff
@@ -15,7 +15,9 @@ const signupGeneralError = error => ({
 export const signupAction = payload => dispatch =>
   signUp(payload)
     .then(() => {
-      const relevantCookies = COOKIE_TYPES.filter(c => c === 'ACCEPT_CGU' || c === 'ACCEPT_PRIVACY_POLICY_ON_DISCUSSION');
+      const relevantCookies = COOKIE_TYPES.filter(
+        c => c === 'ACCEPT_CGU' || c === 'ACCEPT_PRIVACY_POLICY_ON_DISCUSSION' || c === 'ACCEPT_USER_GUIDELINE_ON_DISCUSSION'
```

Comment:
add a constant in the constants file with the items to accept:
export const TO_ACCEPT = ['ACCEPT_CGU' , 'ACCEPT_PRIVACY_POLICY_ON_DISCUSSION', 'ACCEPT_USER_GUIDELINE_ON_DISCUSSION']
Use it like this in this condtion: TO_ACCEPT.includes(c)